test/media_server/movies_index_test.exs

```diff
@@ -0,0 +1,7 @@
+defmodule MediaServer.MoviesIndexTest do
+  use ExUnit.Case
+  
+  test "it should get movie path" do
+    assert MediaServer.MoviesIndex.get_movie_path("1")  === "/movies/Caminandes Llama Drama (2013)/Caminandes.Llama.Drama.1080p.mp4"
```

Comment:
`/movies` would be the "required" path when mounting your movie library into Midarr.

#### Example
`/path/to/movies:/movies`